.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+* @abhinavrastogi-harness @knagurski
```

Comment:
let's remove the `*` rule. 
instead let's add specific root config files like package.json, eslintrc, prettierrc etc

Reply:
Done